src/Amadevus.RecordGenerator.Attributes/RecordAttribute.cs

```diff
@@ -2,9 +2,40 @@
 using System;
 using System.Diagnostics;
 
-[AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = false)]
-[CodeGenerationAttribute("Amadevus.RecordGenerator.Generators.RecordGenerator, Amadevus.RecordGenerator.Generators")]
-[Conditional("CodeGeneration")]
-public sealed class RecordAttribute : Attribute
+namespace Amadevus.RecordGenerator
 {
-}
+    /// <summary>
+    /// This attribute triggers record code generation. <see href="https://amis92.github.io/RecordGenerator/"/>
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = false)]
+    [CodeGenerationAttribute("Amadevus.RecordGenerator.Generators.RecordGenerator, Amadevus.RecordGenerator.Generators")]
+    [Conditional("CodeGeneration")]
+    public sealed class RecordAttribute : Attribute
+    {
+        /// <summary>
+        /// For the annotated type, generate
+        /// <see cref="DefaultRecordFeaturesAttribute.AssemblyDefaultFeatures"/>
+        /// or if not specified, <see cref="Features.Default"/>.
+        /// </summary>
+        public RecordAttribute()
```

Comment:
This should chain to the second constructor rather than rely on property initialisation further down, as in:

```c#
public RecordAttribute() :
    this(Features.Default) {}
```

This is not a question of style but readability and centralised initialisation in the most specific constructor.

Reply:
Good point.

Reply:
Actually, because the assembly-default behavior got dropped, there's no longer any reason to have separate constructors anyway. Merged into one with parameter with default value.